src/js/components/artifacts/artifactPayload.js

```diff
@@ -63,13 +71,13 @@ export const ArtifactPayload = ({ index, payload: { files: payloadFiles, meta_da
                 ))}
               </TableRow>
             </TableHead>
-            <TableBody style={style.table}>
+            <TableBody className={classes.table}>
               {files.map((file, index) => (
                 <TableRow key={index}>
                   <TableCell>{file.name}</TableCell>
                   <TableCell style={{ wordBreak: 'break-word' }}>{file.checksum}</TableCell>
                   <TableCell>
-                    <Time value={file.date} format="YYYY-MM-DD HH:mm" />
+                    <Time value={file.date} />
```

Comment:
this is now moved to a custom time component (since the `react-time` dependency blocked the mui upgrade due to it's limitation to react v16)
[here](https://github.com/mendersoftware/gui/blob/feature/styling-upgrade/src/js/components/common/time.js#L10) the timestamp is set, if you adapt your PR to wrap that custom component with a link to the redux store, you can still inject the format, but yes this change/ PR will require changes to your PR - sorry about that!

Reply:
Alright :)
Thanks